### ISSUE TITLE
Scope resource path extractor doesn't seem to work

### ISSUE THREAD

**Issue by jacobdeichert** [actix/actix-web#234]
It seems that `scope.resource()` is behaving a little differently with path extractors than `app.resource()`.

If I have this extractor function:

```rust
fn example_extractor(path: Path<(u32, String)>) -> HttpResponse {
    println!("info {} {}", path.0, path.1);
    HttpResponse::Ok().finish()
}
```

Using `app.resource()` works:

```rust
app = app.resource("/users/{id}/{name}", |r| {
    r.method(Method::GET).with(example_extractor)
});
// => info 32 jake
```

But using `scope.resource()` does not work:

```rust
app = app.scope("/users", |scope| {
    scope.resource("/{id}/{name}", |r| {
        r.method(Method::GET).with(example_extractor)
    })
});

// => WARN 2018-05-20T15:50:52Z: actix_web::pipeline: Error occured during request handling: can not parse "/32/jake" to a u32
// => DEBUG 2018-05-20T15:50:52Z: actix_web::pipeline: Error { err: "can not parse \"/32/jake\" to a u32" }
```

Am I doing something wrong here? I would really like to use scopes to group top-level endpoints like this. It seems I won't be able to if these path extractors don't work at the scope level.

I'm using v0.6.7. If there's any more details I should provide, let me know!

**Comment by fafhrd91** [comment]
this is bug.

**Comment by fafhrd91** [comment]
fixed by  082ff46
i will release new version tomorrow

**Comment by jacobdeichert** [comment]
That was fast! Thanks 😄
